glide.yaml

```diff
@@ -1,8 +1,7 @@
 package: go.uber.org/fx
 import:
-- package: github.com/uber-go/zap
-  # Pinned temporarily while zap reaches 1.0 (https://github.com/uber-go/fx/pull/243)
-  version: c064b5c44b285a7e2fd5c9b26e8c38228ce2bccb
+- package: go.uber.org/zap
+  version: v1.0.0-rc.1
```

Comment:
Any release timeline on 1.0? We had an incredibly painful amount of churn with yarpc 1.0 RC candidates, both for my team and our consumers.